02-express-tutorial/practiceExpress.js

```diff
@@ -0,0 +1,20 @@
+// In order to make the server work in my package.json I changed the start script to "node practiceExpress.js"
+
+const express = require('express');
+const path = require('path');
+const app = express();
+const port = 3000;
+
+app.use(express.static('./new-public'))
+
+app.get('/sample', (req, res) => {
+  res.send("This is working")
+});
+
+app.all('*', (req, res) => {
+  res.status(404).send('resource not found');
+});
+
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`);
+});
```

Comment:
Very nice!!!

You can also do `npm init -y` inside an empty folder to create a boilerplate `package.json` file. You can then do `npm install express`, create an `index.js` file in that folder, and then you'll be up and running with a fully independent project setup.

Reply:
yes, fully agree. I thought about it too))))